src/uct/tcp/tcp_ep.c

```diff
@@ -625,6 +624,16 @@ static inline ssize_t uct_tcp_ep_send(uct_tcp_ep_t *ep)
     return sent_length;
 }
 
+static inline ssize_t uct_tcp_ep_zcopy_complete_err(uct_tcp_ep_t *ep,
```

Comment:
we should not invoke completion in case we already return error

Reply:
`uct_tcp_ep_sendv()` is only called from the progress function